src/web/wms/src/test/java/org/geoserver/wms/web/data/WMSAdminPageTest.java

```diff
@@ -154,4 +154,14 @@ public void testFeaturesReprojectionDisabled() throws Exception {
         ft.submit("submit");
         assertTrue(wms.isFeaturesReprojectionDisabled());
     }
+
+    @Test
+    public void testIncludeDefaultGroupStyleInCapabilitiesDisabled() throws Exception {
```

Comment:
Holy cow such a long name... due to the property name (see dedicated comment). We are playing with 80 columns, names should be descriptive, yes, but of reasonable length.

```suggestion
    public void testIncludeDefaultGroupStyleInCapabilitiesDisabled() throws Exception {
```

Reply:
fixed